### ISSUE TITLE
Cannot build the core module DB_File

### ISSUE THREAD

**Issue by fperrad** [arsv/perl-cross#20]
On Buildroot, the package `perl` has 2 optional dependencies `berkeleydb` and `gdbm`;
corresponding to the Perl core modules `DB_File` and `GDBM_File`.

With perl-5.22.0 and perl-cross-1.0.1, `gdbm` is correctly detected and the dynamic module `GDBM_File` is built.
But `DB_File` is not built:
```
Looking which extensions should be disabled
	pre-disabling db_file
```

**Comment by arsv** [comment]
https://github.com/arsv/perl-cross/commit/0601282651fc4089b962fc802371e7edd2a0579a

Works for me, but if possible please try it too.
In case everything works, I'll tag 1.0.2.

**Comment by fperrad** [comment]
Works for me.
Thanks.

**Comment by fperrad** [comment]
And Perl 5.22.1is released :)

**Comment by arsv** [comment]
Ok, 1.0.2 released, with that fix and with perl version bumped to 5.22.1.

There are no configure-related changes between 5.22.0 and 5.22.1.
